scenarioo-server/src/test/java/org/scenarioo/business/diffViewer/comparator/ScreenshotComparatorGraphicsMagickTest.java

```diff
@@ -46,13 +48,20 @@ public void setUpClass() throws IOException {
 			getComparisonConfiguration());
 	}
 
+	@After
+	public void cleanUpTest(){
+		if(DIFF_SCREENSHOT.exists()){
+			assertTrue("Unable to clean up the test data: " + DIFF_SCREENSHOT.getAbsolutePath(), DIFF_SCREENSHOT.delete());
+		}
+	}
+
 	@Test
 	public void compareEqualScreenshots() {
-		DIFF_SCREENSHOT.delete();
```

Comment:
I would rather keep to ensure removing it before the test, otherwise this might fail just because the last test run for some reason failed to remove it after the test.